tools/Raw_reads/filter_paired_reads/filter_paired_reads.cwl

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env cwl-runner
+cwlVersion: v1.0
+class: CommandLineTool
+
+label: "remove reads from both files that are less than LEN"
+
+requirements:
+  ResourceRequirement:
+    coresMax: 1
+    ramMin: 200
+
+inputs:
+  forward:
+    type: File
+    format: edam:format_1930
+    inputBinding:
+        prefix: -f
+  reverse:
+    type: File
+    format: edam:format_1930
+    inputBinding:
+        prefix: -r
+  len:
+    type: int
+    inputBinding:
+        prefix: -l
+
+baseCommand: [filter_paired_reads.sh]
+
+outputs:
+  forward_filtered:
+    type: File
+    format: edam:format_1930
+    outputBinding:
+        glob: forward_filt.fastq
```

Comment:
Will this glob output be .gz?

Reply:
No, it would be fastq. SeqPrep accepts as gzipped as regular inputs (but returns compressed anyway).